polymorphism/README.md

```diff
@@ -6,14 +6,20 @@ The $λ^\diamond$-calculus [1], a refined reachability model that scales to para
 
 * [`Base`](lambda_diamond_base) -- Base system introducing the new reachability model, lacking type polymorphism.
 * [`Fsub`](f_sub_diamond) -- Bounded type-and-reachability polymorphism.
+* [`Trans`](f_sub_trans) -- Bounded type-and-reachability polymorphism with explicit transitive closure.
+* [`SelfNat`](f_sub_self_nat) -- Bounded type-and-reachability polymorphism with self references and natural numbers.
```

Comment:
The other versions have self references, too, so better to call this `f_sub_cycles_nat` and talk about cyclic references. Readme should include link to paper (once done).